Preapps 4190

shardool-gore opened the pull request:
Preapps 4190

shardool-gore committed:
New file for z9, changes on packages.adoc

abyshukla committed:
issue #40

shardool-gore committed:
Added conditions for Zimbra 9 install guides

shardool-gore committed:
Changes for Z9

abyshukla commented on licenserequirements.adoc:
```diff
@@ -31,6 +38,10 @@ that can be created. The archive feature must be installed.
 the {product-edition-commercial} of {product-name}.
 
+[IMPORTANT]
+A {product-short} license is required in order to use the {modern} in
```
A {product-short} license is required to use the {modern} in

abyshukla commented on licenserequirements.adoc:
```diff
@@ -31,6 +38,10 @@ that can be created. The archive feature must be installed.
 
+[IMPORTANT]
+A {product-short} license is required in order to use the {modern} in
+the {product-edition-commercial} of {product-name}.
```
{product-name}'s {product-edition-commercial}.

abyshukla commented on upgrade.adoc:
```diff
@@ -29,6 +38,34 @@ and to determine if you wish to install them.
+Please review the following information to decide if {product-name} {product-version}
+is suitable for you.
+
+* Zimbra Touch Client, Zimbra Mobile Client, Zimbra HTML (Standard) Client have been removed from {product-name} {product-version}
```
"have been removed" is passive voice. Please consider:

> Zimbra Touch Client, Zimbra Mobile Client, Zimbra HTML (Standard) Client are no longer a part of {product-name} {product-version}.

abyshukla commented on upgrade.adoc:
```diff
@@ -29,6 +38,34 @@ and to determine if you wish to install them.
+is suitable for you.
+
+* Zimbra Touch Client, Zimbra Mobile Client, Zimbra HTML (Standard) Client have been removed from {product-name} {product-version}
+* {product-name} {product-version} currently does not support Rolling Upgrades
```
full stop at the end
rolling upgrades (lowercase)

abyshukla commented on upgrade.adoc:
```diff
@@ -29,6 +38,34 @@ and to determine if you wish to install them.
+* Zimbra Touch Client, Zimbra Mobile Client, Zimbra HTML (Standard) Client have been removed from {product-name} {product-version}
+* {product-name} {product-version} currently does not support Rolling Upgrades
+* {product-name} {product-version} introduces two versions of {product-short} Web Client, namely, Modern and Classic
+** A {product-short} license is required in order to use the Modern Web Client in
```
"in order to" is too wordy. 'to' is a better replacement.

mogsdad replied:
Change "in order to" -> "to" throughout the document.

abyshukla commented on upgrade.adoc:
```diff
@@ -29,6 +38,34 @@ and to determine if you wish to install them.
+* {product-name} {product-version} currently does not support Rolling Upgrades
+* {product-name} {product-version} introduces two versions of {product-short} Web Client, namely, Modern and Classic
+** A {product-short} license is required in order to use the Modern Web Client in
+the {product-edition-commercial} of {product-name}.
```
> {product-name}'s {product-edition-commercial}.

is crisper instead of 

> the {product-edition-commercial} of {product-name}.

abyshukla commented on upgrade.adoc:
```diff
@@ -29,6 +38,34 @@ and to determine if you wish to install them.
+
+* Zimbra Touch Client, Zimbra Mobile Client, Zimbra HTML (Standard) Client have been removed from {product-name} {product-version}
+* {product-name} {product-version} currently does not support Rolling Upgrades
+* {product-name} {product-version} introduces two versions of {product-short} Web Client, namely, Modern and Classic
```
Please consider using 'em dash' instead of 'namely'
full stop at the end

> * {product-name} {product-version} introduces two versions of {product-short} Web Client -- Modern and Classic.

abyshukla commented on upgrade.adoc:
```diff
@@ -29,6 +38,34 @@ and to determine if you wish to install them.
+the {product-edition-commercial} of {product-name}.
+** To know more about highlights of the Modern Web Client, please refer to the *What's New* document for {product-name} {product-version}
+** Classic Web Client offers the same functionalities as offered by the Advanced Web Client in  {product-name} 8.8.15 version
+** Modern Web Client does not currently support some features like Personas, Briefcase and Dumpster. Some of these features will be added during upcoming releases. To view the detailed comparison of features supported in Modern Web Client, please refer to *Web Client Comparison* document
```
full stop at the end of the sentence. Suggestion
> Modern Web Client does not currently support features like Personas, Briefcase, and Dumpster. Some of these features are part of upcoming releases. To view the detailed comparison of features supported in Modern Web Client, please refer to *Web Client Comparison* document.

abyshukla commented on upgrade.adoc:
```diff
@@ -29,6 +38,34 @@ and to determine if you wish to install them.
+** To know more about highlights of the Modern Web Client, please refer to the *What's New* document for {product-name} {product-version}
+** Classic Web Client offers the same functionalities as offered by the Advanced Web Client in  {product-name} 8.8.15 version
+** Modern Web Client does not currently support some features like Personas, Briefcase and Dumpster. Some of these features will be added during upcoming releases. To view the detailed comparison of features supported in Modern Web Client, please refer to *Web Client Comparison* document
+** Existing customized themes, logo branding changes, and crontab changes will not work with Modern Web Client. Branding needs to be re-configured to work with the Modern Web Client. Themes are currently not supported for Modern Web Client.
```
Future tense and passive voice is avoided in technical writing. Suggestion
> ** Existing customized themes, logo branding changes, and crontab changes are incompatible with, and hence do not reflect in, the Modern Web Client. Branding needs to be re-configured to work with the Modern Web Client. The web client also does not currently support themes.

abyshukla commented on upgrade.adoc:
```diff
@@ -29,6 +38,34 @@ and to determine if you wish to install them.
+** Modern Web Client does not currently support some features like Personas, Briefcase and Dumpster. Some of these features will be added during upcoming releases. To view the detailed comparison of features supported in Modern Web Client, please refer to *Web Client Comparison* document
+** Existing customized themes, logo branding changes, and crontab changes will not work with Modern Web Client. Branding needs to be re-configured to work with the Modern Web Client. Themes are currently not supported for Modern Web Client.
+** Zimlets are supported on both the Web Clients. 
+*** Existing Zimlets will continue to work in the Classic Web Client, however they will not work on Modern Web Client. There is no way to migrate the zimlets from Classic to Modern Web Client or vice-versa. 
```
Future tense is avoided in technical writing. Suggestion
> *** Existing Zimlets work with Classic Web Client; however, they are incompatible with Modern Web Client. Currently, there is no way to migrate the zimlets from Classic to Modern Web Client or vice-versa.

abyshukla commented on upgrade.adoc:
```diff
@@ -29,6 +38,34 @@ and to determine if you wish to install them.
+** Existing customized themes, logo branding changes, and crontab changes will not work with Modern Web Client. Branding needs to be re-configured to work with the Modern Web Client. Themes are currently not supported for Modern Web Client.
+** Zimlets are supported on both the Web Clients. 
+*** Existing Zimlets will continue to work in the Classic Web Client, however they will not work on Modern Web Client. There is no way to migrate the zimlets from Classic to Modern Web Client or vice-versa. 
+*** Zimbra Connect zimlet, Zimbra Drive zimlet and Zimbra Docs zimlet, are available for both the web clients. Seperate versions of these services need to be installed for Classic Web Client and Modern Web Client
```
passive voice. Suggestion
> *** Zimbra Connect zimlet, Zimbra Drive zimlet, and Zimbra Docs zimlet, are available for both the web clients. You can install separate versions of these services for Classic Web Client and Modern Web Client.

abyshukla commented on upgrade.adoc:
```diff
@@ -29,6 +38,34 @@ and to determine if you wish to install them.
+*** Existing Zimlets will continue to work in the Classic Web Client, however they will not work on Modern Web Client. There is no way to migrate the zimlets from Classic to Modern Web Client or vice-versa. 
+*** Zimbra Connect zimlet, Zimbra Drive zimlet and Zimbra Docs zimlet, are available for both the web clients. Seperate versions of these services need to be installed for Classic Web Client and Modern Web Client
+
+The following tasks might need to be performed before you
```
passive voice. Suggestion
> You might need to perform the following tasks before the

abyshukla commented on upgrade.adoc:
```diff
@@ -520,14 +557,26 @@ change your configuration.  Configuration settings stored in _LDAP_ and
 
+ifdef::z9[]
+[IMPORTANT]
+Zimlets are supported on both Classic and Modern Web Clients. However, Zimlets used for Classic Web Client are not compatible with Modern Web Client. There is no way to migrate the zimlets from Classic to Modern Web Client or vice-versa. Existing zimlets need to be re-developed in order for them to work on Modern Web Client.
```
Repeated

abyshukla commented on upgrade.adoc:
```diff
@@ -520,14 +557,26 @@ change your configuration.  Configuration settings stored in _LDAP_ and
+
+ifdef::z9[]
+[IMPORTANT]
+Branding needs to be re-configured to work with the Modern Web Client. Themes are currently not supported on Modern Web Client.
```
Repeated.

shardool-gore replied:
we need the repetition here.

abyshukla commented on upgrade.adoc:
```diff
@@ -29,6 +38,34 @@ and to determine if you wish to install them.
+*** Zimbra Connect zimlet, Zimbra Drive zimlet and Zimbra Docs zimlet, are available for both the web clients. Seperate versions of these services need to be installed for Classic Web Client and Modern Web Client
+
+The following tasks might need to be performed before you
+upgrade. After you review the tasks in this section, go to Upgrade
```
"please"
> please  go to Upgrade

abyshukla commented on licenserequirements.adoc:
```diff
@@ -14,6 +14,13 @@ the number of accounts created are shown.
+ifdef::z9[]
++
+[IMPORTANT]
+Mobile and Touch clients have been removed from {product-name} {product-version}.
```
passive voice. Suggestion
> Mobile and Touch clients are no longer a part of {product-name} {product-version}.

shardool-gore replied:
agree. changed

shardool-gore committed:
Review comments addressed

shardool-gore committed:
Optimized code

shardool-gore committed:
Updated review

shardool-gore committed:
review comments

shardool-gore committed:
Review

shardool-gore committed:
Beta

abyshukla committed:
Updated Before you upgrade and installation switch

Commit:
Dead documentation cleanup

- Removed outdated Zimbra-X content
- Removed deprecated IMAPD content

Commit:
Update version numbers (8.8.15 & 9.0.0)

Commit:
Changed "modern" variable to a noun "modern-client"

Commit:
Update systemrequirements.adoc

- Refreshed PC & Server specs
- Refreshed OS & Application versions

Commit:
Applied web-client variables throughout

mogsdad committed:
Merge branch 'develop' into preapps-4190

Commit:
Updated CPU examples with current models

abyshukla commented on systemrequirements.adoc:
```diff
@@ -48,24 +48,23 @@ additional space for mail storage. The other nodes require 100MB.
-* Ubuntu 16.04 LTS Server Edition (64-bit), starting from Zimbra Collaboration 8.7.1 and above
-* Ubuntu 18.04 LTS Server Edition (64-bit), starting from Zimbra Collaboration 8.8.15 and above
+* Ubuntu 16.04 LTS Server Edition (64-bit)
+* Ubuntu 18.04 LTS Server Edition (64-bit), starting from Zimbra Collaboration 8.8.12 and above
```
Ubuntu 18.04 was GA with 8.8.15. With 8.8.12, it was still beta.

Commit:
In Z9 we will start calling the clients "Web Apps".

- This will be more familiar to users, as "Apps" have entered mainstream lexicons globally.
- "Web app" translates accurately, whereas "Client" is interpreted as "Customer" by machine translation, in some languages.
